package.json

```diff
@@ -45,6 +45,7 @@
   },
   "dependencies": {
     "@popperjs/core": "^2.9.2",
+    "@types/react-datepicker": "^4.1.5",
```

Comment:
Any reason for moving it here? Usually, typings are dev dependencies as they are not required at runtime. Maybe have them as `peerDependencies`? Then they would still be automatically resolved if someone does not have them, but they are not indicated as required per se.

Reply:
You are right. But I need it not at runtime, I need it in development (when use `DateField` component). When it's in `dependencies` - it's not a part of precise-ui installed into zeisslet deps.

Reply:
Yes, but that is not a problem of precise-ui. If every library would need to care about the dev setup of any consumer app nothing would ever ship.

The solution is that the Zeisslet needs to install the right typings. After all, it seems to care about these typings (for some reason). Otherwise, I'd recommend not to expose third-party typings.